.github/workflows/ci.yml

```diff
@@ -204,6 +204,13 @@ jobs:
       id-token: write
       contents: read
     needs: [install-dependencies]
+    if: |
+
+      github.base_ref == 'master' ||  contains(github.base_ref, 'release') ||
+      contains(needs.install-dependencies.outputs.all_modified_files, 'packages/sdk') ||
+      contains(needs.install-dependencies.outputs.all_modified_files, ',package.json') ||
+      contains(needs.install-dependencies.outputs.all_modified_files, ',yarn.lock') ||
```

Comment:
if i recall @shazarre removed these because not running the tests each time messes up the code cov reports